eyeshade/workers/reports.js

```diff
@@ -1240,7 +1240,8 @@ exports.workers = {
             publisher: slice.publisher,
             votes: slice.counts,
             created: new Date(parseInt(slice._id.toHexString().substring(0, 8), 16) * 1000).getTime(),
-            modified: (slice.timestamp.high_ * 1000) + (slice.timestamp.low_ / bson.Timestamp.TWO_PWR_32_DBL_)
+            modified: (slice.timestamp.high_ * 1000) + (slice.timestamp.low_ / bson.Timestamp.TWO_PWR_32_DBL_),
+            cohort: slice.cohort || 'control'
```

Comment:
this is why i don't like the `standard` rule for where to put commas between properties. if we could put them at the beginning of the line, then the only diff here would be the line `, cohort: ...`

Reply:
something, something trailing commas :)

Reply:
_heh._ however, there is a serious point here: i'd rather see one green line and not a red line followed by two green ones, where the only difference between the red and first green line is the addition of a comma. it is visually distracting otherwise.

Reply:
:P yeah i'm agreeing that a single line change would be better, I just wish it was via a trailing comma after the final element instead of leading comma before each element

Reply:
understood, i think the issue is that seeing a comma at EOL suggests there should be something on the next line. seeing a comma and then text disambiguates that...